style.css

```diff
@@ -102,6 +102,46 @@ a {
 
 .primary-menu > li {
 	margin-left: 2rem;
+  
+/*--------------------------------------------------------------
+# Header
+--------------------------------------------------------------*/
+
+.site-description {
+	display: none;
+}
+
+.cover-header {
+	background-color: var( --color-dark-alt-text );
+}
+
+.cover-header .cover-header-inner-wrapper {
+	justify-content: center;
+	min-height: auto;
+	height: 320px;
+	max-height: 100vh;
+}
+
+.admin-bar .cover-header-inner-wrapper {
+	max-height: calc( 100vh - 32px );
+}
+
+.home .cover-header-inner-wrapper {
+	height: 450px;
```

Comment:
I see that the comps suggest this 450px value, but if I'm interpreting the design, it reads like the home page cover image should take over the whole viewport. Also because of the down scroll arrow UI.

<img width="418" alt="Screen Shot 2020-04-26 at 4 48 04 PM" src="https://user-images.githubusercontent.com/5375500/80319392-d89e5a00-87dd-11ea-83d3-10a96b01ec9b.png">